### PR TITLE
Register Windows native artifacts in release API

### PR DESCRIPTION
**Pull request by justinas**
Contributes to: https://github.com/gravitational/cloud/issues/2103

This utilizes a static binary of [relcli](https://github.com/gravitational/cloud/tree/master/scripts/relcli), fetched from the CDN, to automatically scan the outputs of the build and register the artifacts (currently, only Teleport Connect windows installer) in the release API.

### CODE REVIEW

File: dronegen/windows.go [hunk 1]
```diff
@@ -23,6 +23,9 @@ const (
 	toolchainDir      = `/toolchains`
 	teleportSrc       = `/go/src/github.com/gravitational/teleport`
 	webappsSrc        = `/go/src/github.com/gravitational/webapps`
+
+	relcliUrl    = `https://cdn.cloud.gravitational.io/relcli-windows-v1.1.70-justinas.1.exe`
```
Comment (justinas) [inline dronegen/windows.go:27]: These are placeholder values (a release created manually on staging) while we work out the relcli windows publishing pipeline on the Cloud side.

**Review by tcsc — APPROVED** [review]
LGTM

File: .drone.yml [hunk 2]
```diff
@@ -936,6 +936,25 @@ steps:
...
       from_secret: AWS_SECRET_ACCESS_KEY
     WORKSPACE_DIR: C:/Drone/Workspace/build-native-windows-amd64
+- name: Register artifacts
+  commands:
+  - $ErrorActionPreference = 'Stop'
+  - $ProgressPreference = 'SilentlyContinue'
+  - $Workspace = "$Env:WORKSPACE_DIR/$Env:DRONE_BUILD_NUMBER"
+  - $TeleportSrc = "$Workspace/go/src/github.com/gravitational/teleport"
+  - $OutputsDir = "$Workspace/outputs"
+  - $relcliUrl = 'https://cdn.teleport.dev/relcli-v1.1.70-beta.1-windows.exe'
```
Comment (justinas) [inline .drone.yml:946]: Final v1.1.70 will likely get cut this Thursday. I plan to bump the version with the upcoming PR that will also switch to using the production releases instance. No relcli changes are expected.

File: dronegen/windows.go [hunk 3]
```diff
@@ -23,6 +23,9 @@ const (
 	toolchainDir      = `/toolchains`
 	teleportSrc       = `/go/src/github.com/gravitational/teleport`
 	webappsSrc        = `/go/src/github.com/gravitational/webapps`
+
+	relcliURL    = `https://cdn.teleport.dev/relcli-v1.1.70-beta.2-windows.exe`
```
Comment (justinas) [inline dronegen/windows.go:27]: Final v1.1.70 will likely get cut this Thursday. I plan to bump the version with the upcoming PR that will also switch to using the production releases instance. No further relcli changes are expected.

**Review by ibeckermayer — APPROVED** [review]

**Review by espadolini — APPROVED** [review]

**Pull request merged**